src/containers/education/Education.css

```diff
@@ -0,0 +1,46 @@
+.education-heading {
+  font-size: 56px;
+  font-weight: 400;
+}
+
+.education-section {
+  display: flex;
+  width: 90%;
+  padding: 20px 10px;
+  margin: 0px auto;
+  margin-top: 2rem;
+}
+
+.education-section > * {
+  flex: 1;
+  margin-bottom: 30px;
+}
+
+.education-cards {
```

Comment:
My bad earlier, to make it a bit different, we can go for `education-card-container` or something of the sort. What do you think?